### PR TITLE
Feature/banner section

### PR DESCRIPTION
**Pull request by vinayak-4**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,5 +50,14 @@
         </ul>
       </nav>
     </header>
+    <main>
+      <section id="banner">
+        <div class="banner__text">
+          <h6>Welcome To Cyborg</h6>
```
Comment (Delwalt) [inline index.html]: heading should follow the correct order. H6 should not come before h4. Think HTML page as a word document.

File: style.css [hunk 2]
```diff
@@ -115,14 +117,70 @@ header > i {
...
+
+.primary-btn {
+  background: var(--primary-color);
+  color: var(--off-white);
+  border: none;
+  outline: none;
+  padding: 12px 27px;
+  border-radius: 25px;
+  cursor: pointer;
+  font-size: .9rem;
```
Comment (Delwalt) [inline style.css:167]: Make sure you are using REM unit font sizes correctly otherwise this can create issues on responsiveness

**Review by Delwalt — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (vinayak-4) [reply index.html]: Fixed it

File: style.css (hunk 2 above)
Comment (vinayak-4) [reply style.css:167]: I want the font size of the button 15px and the root font size is 16px that's why I used 0.9 rem.

**Review by Delwalt — APPROVED** [review]

**Pull request merged**